packages/jest-circus/src/legacy-code-todo-rewrite/jestExpect.js

```diff
@@ -27,15 +27,16 @@ type JasmineMatcher = {
 
 export default (config: {expand: boolean}) => {
   global.expect = expect;
-  expect.setState({
-    expand: config.expand,
-  });
-  expect.extend({
+  setState({expand: config.expand});
+  extend({
     toMatchInlineSnapshot,
     toMatchSnapshot,
     toThrowErrorMatchingInlineSnapshot,
     toThrowErrorMatchingSnapshot,
   });
+  global.expect.extend = extend;
+  global.expect.assertions = assertions;
+  global.expect.hasAssertions = hasAssertions;
```

Comment:
Aren't these methods already bound to `expect`? There's also `expect.any`, `expect.anything`, maybe it's better to `import * as expect from 'expect';`?

Reply:
I changed this back to how it was.